rest/netutil.go

```diff
@@ -130,11 +130,15 @@ func (c *Client) RestAPICall(method Method, path string, options interface{}) ([
 			return nil, err
 		}
 		log.Debugf("*** options => %+v", bytes.NewBuffer(OptionsJSON))
+		ioutil.WriteFile("jsons.json", OptionsJSON, 0644)
```

Comment:
Are these statements required?

Reply:
It was for debugging purpose. Now I have removed.